process/block/metablock.go

```diff
@@ -969,6 +970,8 @@ func (mp *metaProcessor) CommitBlock(
 	)
 
 	lastMetaBlock := chainHandler.GetCurrentBlockHeader()
+	mp.updateAccountsStateStorage(lastMetaBlock, mp.stateCheckpointModulus)
```

Comment:
Maybe create a function for these 2 instructions as to keep them organized

Reply:
Done.